connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -1737,7 +1721,7 @@ export async function renderAndUpsertPageFromCache({
   const createdAt = new Date(pageCacheEntry.createdTime);
   const updatedAt = new Date(pageCacheEntry.lastEditedTime);
 
-  if (!pageHasBody) {
+  if (documentLength === 0) {
```

Comment:
@fontanierh sanity check: pageHasBody flag replaced by computing parsed content length, seems ok but wanted to check with you in case the flag had a specific importance